index.html

```diff
@@ -4,7 +4,140 @@
+              </li>
+              <li>
+                <h3>13.12.04</h3>
+                Stephan Geene
+                <span>Marchel Broodthaers — tapes</span>
+              </li>
+            </ul>
+          </div>
+        
+        <div class="item">
+            <h2>Trichtlinnburg</h2>
+            <h3>12—13.11.04</h3>
+            <p class="indent-event-description">Trichtlinnburg, a new city, making its first appearance, feat. the Goldextra Djvj-Sightclub and Hip Hop on Bones by Kinematix</p>
+            <div class="em-dash-container">
+              <div class="em-dash"></div>
+                <p>From 9 pm till 1 am</p>
+            </div>
+                <p class="indent-event-description">Cafe Zuid<br>Plein 1992, 15<br>Wyck/Ceramique<br>Maastricht</p>
+        </div>
+  
+          <div class="item">
+            <h3>12—13.11.04</h3>
+            <p>Benjamin Buchloh, Wouter Davidts, Stijin Huijts, Christian Kravagna, Johanne Lamoureux, Allan Wallach, John Welchman, Camiel Van Winkel</p>
+            <h2 class="indent-event-description">Museum in Motion?</h2>
+            <div class="em-dash-container">
+              <div class="em-dash"></div>
+              <p>conference</p>
+            </div>
+              <p class="indent-event-description">[with department of architecture & urban planning, ghent university & museum het domen, sittaro]</p>
+          </div>
+  
+          <div class="item">
+            <h4>Jan van Eyck Academie</h4>
+            <ul>
+              <li>Post-academic institute for research and production</li>
+              <li>Fine art, design, theory</li>
+            </ul>
+          </div>
+
+          <div class="item">
+            <ul>
+            <li>
+              <address>Academieplein 1<br>6211 KM maastricht<br>The Netherlands<br>T +31 (0)43 350 37 37</address>
+            </li>
+          </ul>
+          </div>
+  
+        <div class="item">
+            <h2>Authoring the City</h2>
+            <h3>25—26.11.04</h3>
+            <div class="em-dash-container">
+              <div class="em-dash"></div>
+              <p>lectures, presentations</p>
+            </div>
+        </div>
+  
+        <div class="item">
+          <h3>10.12.04</h3>
+          <h2>Letterror</h2>
+          <p class="large-font">[just van rossum, erik van blokland], charles esche, lars muller, walter nikkels, rick poynor, etc.</p>
+          <p class="indent-event-description">charles nypels lectures</p>
+          <div class="em-dash-container">
+            <div class="em-dash"></div>
+            <p>conference</p>
+          </div>
+        </div>
+  
+        <div class="item">
+          <h4>Events</h4>
+            <ul>
+              <li>
+                For more information: <a href="https://www.janvaneyck.nl/" target="_blank">www.janvaneyck.nl</a>
+              </li>
+              <li>
+                Subscription mailing list: program@janvaneyck.nl
```

Comment:
You could also make this email address a link like so:
```suggestion
                Subscription mailing list: <a href="mailto: program@janvaneyck.nl">program@janvaneyck.nl</a>
```